cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -88,6 +89,23 @@ function computeZRange(states: any[]): number[] {
     return [minZ, maxZ];
 }
 
+async function jobInfoGenerator(job: any): Promise<Record<string, number>> {
+    const { total } = await job.annotations.statistics();
+    return {
+        'frame count': job.stopFrame - job.startFrame + 1,
+        'track count': total.rectangle.shape + total.rectangle.track
+            + total.polygon.shape + total.polygon.track
+            + total.polyline.shape + total.polyline.track
+            + total.points.shape + total.points.track,
```

Comment:
@azhavoro 
Could you please look at this code?
I did it like in the previous version, but I am not sure it's right
I tell about ``track count``